README.md

```diff
@@ -39,7 +39,7 @@ For authentication, you must provide an API key that you will find in your accou
 use Clearhaus\Client;
 
 $client = new Client();
-$client->setApiKey($apiKey);
+$client->useApiKey($apiKey);
```

Comment:
Thinking about it, wouldn't it be better to just have the api key as constructor parameter? You can't really do any call with no api key anyway...